cfgov/v1/util/util.py

```diff
@@ -64,6 +66,11 @@ def get_secondary_nav_items(request, current_page):
             page.get_appropriate_siblings()
         )
 
+    if wagtail.VERSION > (2, 8):
```

Comment:
```suggestion
    if wagtail.VERSION >= (2, 9):
```

Wouldn't this line erroneously get hit on versions 2.8.1 or 2.8.2? Alternatively you could switch the logic around and use `if wagtail.VERSION < (2, 9):` as above.

Reply:
Also, to eliminate the coverage failure, you can add a `  # pragma: no cover` to this test (coverage is checked on `unittest-current`).

Reply:
Updated to eliminate coverage failure and changed > (2, 8) to < (2, 9) with logic switch in `util.py`